### DISCUSSION TITLE
What is the difference between zotero://note/u/ID/ and zotero://select/library/items/ID ?

### DISCUSSION THREAD

**Discussion by ulahcherubim** [windingwind/zotero-better-notes#900] [Q&A] [answered]
I have a question regarding the IDs which are created while adding a new note from an annotation via Zotero Better Notes. When a new note is created via this function it assigns the annotation with a tag zotero://note/u/ID/ which is different from zotero://select/library/items/ID which is created via Actions and Tags plugin's copy item link function. The question is, are there any differences between theese two links, and which one to use is better for the longevity? Thanks for the clarifcations beforehand.

After testing wiht various scenarios, I figured out that the zotero://note/u/ID/ is for opening a note in note workspace of ZBN plugin. So, using these links to maintain access to notes is not a good idea for the longevity, I think i will be using links created by Actions and Tags plugin.

**Comment by windingwind** [comment, marked as answer]
`zotero://select` select item in the library;
`zotero://note` is note link supported by this plugin, which opens the note, similar to `zotero://open-pdf`

**Reply by ulahcherubim** [reply]
Confirms what I think, thank you!
